smalivm/src/test/java/org/cf/smalivm/TestMethodInheritence.java

```diff
@@ -15,10 +15,10 @@
     public static class ChildClass {
         private static final String CLASS_NAME = "Lchild_class;";
         private static final String EXPECTED_VALUE = "abstract?";
-        private VirtualMachine vm;
+        private static VirtualMachine vm;
 
-        @Before
-        public void setupVM() {
+        @BeforeClass
```

Comment:
Fair enough - it seemed to be OK here, though I get what you mean and it could likely be a bad precedent.